### PR TITLE
migration for peerInformation table

### PR DESCRIPTION
**Pull request by penk85**

### CODE REVIEW

File: packages/api/prisma/schema.prisma [hunk 1]
```diff
@@ -686,3 +689,16 @@ model Setting {
 
   @@map("settings")
 }
+
+model ArticlePeerInformation {
```
Comment (Itrulia) [inline packages/api/prisma/schema.prisma:693]: Do we want to have the information here if the article has been edited?

File: packages/api/prisma/schema.prisma [hunk 2]
```diff
@@ -686,3 +689,16 @@ model Setting {
...
+  createdAt  DateTime @default(now())
+  modifiedAt DateTime @updatedAt
+
+  peerId String
+  peer   Peer   @relation(fields: [peerId], references: [id])
+
+  producerArticleId String
+  consumerArticleId String  @unique
+  consumerArticle   Article @relation(fields: [consumerArticleId], references: [id])
+}
```
Comment (Itrulia) [inline packages/api/prisma/schema.prisma:704]: Missing `@@map('articles.peer-information')` to set the table name. Else the table name is `ArticlePeerInformation`

**Review by Itrulia — DISMISSED** [review]

File: packages/api/prisma/migrations/20220905153523_peering/migration.sql [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    "isEdited" BOOLEAN NOT NULL DEFAULT false,
+    "peerId" TEXT NOT NULL,
+    "producerArticleId" TEXT NOT NULL,
+    "consumerArticleId" TEXT NOT NULL,
+
+    CONSTRAINT "articles.peer-information_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "articles.peer-information_producerArticleId_key" ON "articles.peer-information"("producerArticleId");
```
Comment (elias-summermatter) [inline packages/api/prisma/migrations/20220905153523_peering/migration.sql:15]: Shouldn't that be combined unique constraint with peerId?

**Review by elias-summermatter — CHANGES_REQUESTED** [review]

**Comment by tomaszdurka** [comment]
Closing in favor of https://github.com/wepublish/wepublish/pull/734

**Pull request closed without merging**